packages/core/upload/admin/src/hooks/useAssets.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { useNotifyAT } from '@strapi/parts/LiveRegions';
+import { useQuery as useURLQuery, useNotification } from '@strapi/helper-plugin';
+import { useIntl } from 'react-intl';
+import { axiosInstance, getRequestUrl, generateStringFromParams } from '../utils';
+import useSelectTimestamps from './useSelectTimestamps';
+
+export const useAssets = ({ skipWhen }) => {
+  const { formatMessage } = useIntl();
+  const toggleNotification = useNotification();
+  const [, updated_at] = useSelectTimestamps();
+  const { notifyStatus } = useNotifyAT();
+  const query = useURLQuery();
+  const dataRequestURL = getRequestUrl('files');
+  const params = generateStringFromParams(query);
+  const paramsToSend = params.includes('sort') ? params : params.concat(`&sort=${updated_at}:DESC`);
+
+  const { data, error, isLoading } = useQuery(
+    'assets',
```

Comment:
```
['assets', rawQuery]
```

Reply:
why?

Reply:
According to the docs it's easier to make the pagination, you can prefetch the next page and the page before this way.

Reply:
Let's to this in the pagination PR if it's okay for you